.github/workflows/init_repo.yml

```
@@ -11,6 +11,16 @@ jobs:
           fetch-depth: 0
           ref: ${{ github.head_ref }}
 
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install bits
```

Why not this, or some variation?

```suggestion
          pip install -e .
```